apps/frontpage/next.config.js

```diff
@@ -74,6 +74,11 @@ module.exports = {
     return config;
   },
   transpilePackages: ['@repo/ui', '@repo/utils'],
+  experimental: {
```

Comment:
See: https://vercel.com/guides/how-can-i-use-files-in-serverless-functions#using-next.js